<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddPersonDialog</class>
 <widget class="QDialog" name="AddPersonDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>456</width>
    <height>232</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Add New Person</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="labelFirstName">
       <property name="text">
        <string>Imię</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="firstNameEdit"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="labelLastName">
       <property name="text">
        <string>Nazwisko</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="lastNameEdit"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="labelBirthDate">
       <property name="text">
        <string>Data urodzenia</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <layout class="QHBoxLayout" name="birthDateLayout">
       <item>
        <widget class="QLabel" name="labelYear">
         <property name="text">
          <string>Year:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="yearSpin">
         <property name="minimum">
          <number>1900</number>
         </property>
         <property name="maximum">
          <number>2100</number>
         </property>
         <property name="value">
          <number>1990</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="labelMonth">
         <property name="text">
          <string>Month:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="monthSpin">
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>12</number>
         </property>
         <property name="value">
          <number>1</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="labelDay">
         <property name="text">
          <string>Day:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="daySpin">
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>31</number>
         </property>
         <property name="value">
          <number>1</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="3" column="0">
      <widget class="QCheckBox" name="actorCheckBox">
       <property name="text">
        <string>Aktor</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QCheckBox" name="directorCheckBox">
       <property name="text">
        <string>Reżyser</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Orientation::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::StandardButton::Cancel|QDialogButtonBox::StandardButton::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>AddPersonDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>248</x>
     <y>254</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>AddPersonDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>316</x>
     <y>254</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
